.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Prettier's config file. Prettier is a godsend that makes all your code look beautiful automagically (just make sure to turn on "format on save" in your VSCode settings... in `settings.json` you want to add `"editor.formatOnSave": true,`)